geomstats/geometry/embedded_manifold.py

```diff
@@ -9,28 +9,64 @@ class EmbeddedManifold(Manifold):
     """Class for manifolds embedded in another manifold."""
 
     def __init__(self, dimension, embedding_manifold):
+        """Construct the EmbeddedManifold object.
```

Comment:
Shouldn't the constructor be documented as part of the class docstring? See https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes.

Reply:
Yes! Thanks for catching this! It's updated here and in the other files.